package.json

```diff
@@ -12,13 +12,14 @@
 		"url": "git+https://github.com/bitfocus/companion-module-generic-dataentry.git"
 	},
 	"dependencies": {
-		"@companion-module/base": "~1.5.1",
+		"@companion-module/base": "^1.13.2",
```

Comment:
Just noting, that updating this to 1.13 will require users to be on Companion 4.1, which just came out recently.
https://github.com/bitfocus/companion-module-base?tab=readme-ov-file#supported-versions-of-this-library

It might be worth making this 1.12 so that 4.0 users can get the fix